widgets/tooltip_list.py

```diff
@@ -18,6 +18,12 @@
     "No Mushroom Zone": "Whether CPUs are allowed to use mushrooms at this point or not",
 }
 
+checkpoints = {
+    "Shortcut Point ID": "Halts the lap checker on all regions containing checkpoint pairs with this ID, allowing for dev-intended shortcuts.\nOnce the region is left, the lap checker will jump to the next standard checkpoint.",
+    "Double-sided": "If set, the game will check if this point has been crossed from the opposite side.",
+    "Lap Checkpoint": "(Will only work with the \"Sectioned Courses\" patch enabled) \nWill increment a lap when crossed. \nIf this is the last checkpoint of its kind, it will finish the race instead. \nIf \"Shortcut Point ID\" is set, this flag is ignored.",
```

Comment:
In my work-in-progress changes for the Type-specific Item boxes code patch, I went for **Item type (*)** for the label of the fourth field, hoping that the `(*)` triggers the user to check the tool tip. Thoughts?

Reply:
Seems like a good enough idea to try.